.rubocop.yml

```diff
@@ -0,0 +1,118 @@
+---
+require: rubocop-rspec
```

Comment:
We didn't enforce Rubocop linting before?

Makes sense to include that as a new improvement in CHANGELOG.

Reply:
Done!